data/card_definitions.json

```diff
@@ -6852,383 +6852,1036 @@
 
 
 	{
-		"NOTE": "ZATO CARDS START",
-		"id": "zato_sunvoid",
+		"NOTE": "YUZURIHA CARDS START",
+		"id": "yuzu_kashou",
 		"type": "ultra",
-		"display_name": "Sun Void",
+		"display_name": "Sogetsu Ittou-Ryuu Ougi: Kashou",
 		"force_cost": 0,
 		"gauge_cost": 2,
-		"range_min": 0,
-		"range_max": 1,
-		"speed": 7,
-		"power": 3,
+		"range_min": 1,
+		"range_max": 4,
+		"speed": 6,
+		"power": 0,
 		"armor": 0,
 		"guard": 0,
 		"effects": [
 			{
 				"timing": "during_strike",
-				"effect_type": "do_not_remove_eddie"
+				"effect_type": "cannot_stun"
+			},
+			{
+				"timing": "hit",
+				"effect_type": "set_strike_x",
+				"source": "top_discard_power",
```

Comment:
is it obvious in the UI how this ends up getting chosen? I suspect not. I'd like to try to add something, or maybe just auto sort the spent gauge to pick the highest between them so the user doesn't even have to worry about it?

Reply:
As in making it obvious which gauge card will end up at the top of the discard pile? I can try adding in special reminder text and maybe a different highlight color; hesitant to auto-sort gauge in case someone's trying to play around some weird interaction (e.g. mirror of the world)

Reply:
Yea, maybe for now just something "Cards discarded in order selected" or something? I dunno. You can try whatever works good in the UI.